### PR TITLE
PLANET-5443 Trigger rollback on failed tests

### PR DESCRIPTION
**Pull request by comzeradd**

### CODE REVIEW

**Review by lithrel — APPROVED** [review]
:+1:

File: trigger_rollback.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  --arg RELEASE_STAGE "rollback" \
+'{
+  "tag": $VAL,
+  "parameters": {
+    "build_num": $BUILD_NUM,
+    "release_stage": $RELEASE_STAGE
+  }
+}')
+
+echo "Build: ${user}/${repo}@${tag}"
```
Comment (Inwerpsel) [inline trigger_rollback.sh:20]: We echo here that it's triggering a rollback, currently the job output doesn't specify say it's triggering a rollback.

Comment (Inwerpsel) [reply trigger_rollback.sh:20]: Maybe we can echo the json of the request too?

Comment (comzeradd) [reply trigger_rollback.sh:20]: Good point. I added a couple of `echo` outputs there to make it more verbose.

**Pull request merged**